src/main/java/com/training/assignments/Assignment2.java

```diff
@@ -0,0 +1,8 @@
+package com.training.assignments;
+
+public class Assignment2 {
+    public String string1(String word1, String word2) {
```

Comment:
give meaningful Variable name ie newName,oldName,and use prefix get in method as it is returning string value
give method name like stringReplace